README.md

```diff
@@ -85,96 +169,66 @@ And check with `FeatureService.enabled?("some.nested_feature")`.
 
 Rspec tests can also be tagged with `feature_{name}: true`. This will turn that feature on just for the duration of that test.
 
-### Running the app
-
-You can either run the development task:
-
-```bash
-# Run the foreman dev server. This will also start the frontend dev task
-./bin/dev
-```
+## Configuring GOV.UK Notify
 
-or run the rails server:
+We use [GOV.UK Notify] to send emails from our apps.
 
-```bash
-# Run a local Rails server
-bundle exec rails server
+If you want to test the Notify functionality locally, you will need to get a test API key from the Notify service. Add it as an environment variable under `SETTINGS__GOVUK_NOTIFY__API_KEY` or add it to a local config file:
 
-# When running the server, you can use any of the frontend tasks, e.g.:
-npm run dev
 ```
+# config/settings.local.yml
 
-### Running the tests
-
-```bash
-bundle exec rake
+# Settings for GOV.UK Notify api & email templates
+govuk_notify:
+  api_key: <API key from Notify>
 ```
 
-## Configuration and deployment
+Example emails can be seen locally by visiting `http://localhost:3000/rails/mailers`
 
-The forms-admin app is containerised (see [Dockerfile](https://github.com/alphagov/forms-admin/blob/main/Dockerfile)) and can be deployed however you would normally deploy a containerised app.
+[GOV.UK Notify]: https://www.notifications.service.gov.uk/
 
-If you are planning to deploy to GOV.UK PaaS without using the container, you can see how this runs in our [Deployment CI action](https://github.com/alphagov/forms-admin/blob/main/.github/workflows/deploy.yml).
+## Configuring Sentry
 
-## Logging
+We use [Sentry] to catch exceptions in production apps and alert us.
 
-- HTTP access logs are managed using [Lograge](https://github.com/roidrage/lograge) and configured within [the application config](./config/application.rb)
-- The output format is JSON using the [JsonLogFormatter](./app/lib/json_log_formatter.rb) to enable simpler searching and visbility especially in Splunk.
-- **DO NOT** use [log_tags](https://guides.rubyonrails.org/configuring.html#config-log-tags) since it breaks the JSON formatting produced by Lograge.
+We currently have a very basic setup for Sentry in this repo for testing, which we will continue to build upon.
 
-## Explain how to test the project
+In order to use this locally you will need to sign up to Sentry and create a new project. Then add the Sentry DSN to your environment as `SETTINGS__SENTRY__DSN`, or add it to a local config file:
```

Comment:
Could we use an ordered list here to make it easier to read?

In order to use Sentry locally, you will need to:

1. Sign up to Sentry (could we link to an explanation of how to do this?)
2. Create a new project.
3. Add the Sentry DSN to your environment as `SETTINGS__SENTRY__DSN`, or add it to a local config file:

Reply:
I think in most cases devs in our team will already have a Sentry account, so point one could be:

> 1. Sign in to Sentry using your work Google account

What do you think?